arangod/Aql/IResearchViewExecutor.h

```diff
@@ -562,7 +547,7 @@ class IResearchViewExecutorBase {
   std::vector<ColumnIterator> _storedValuesReaders;
   std::array<char, arangodb::iresearch::kSearchDocBufSize> _buf;
   bool _isInitialized;
-
+  std::vector<float_t> _singleDocScoreBuffer;
```

Comment:
```suggestion
  std::vector<float> _singleDocScoreBuffer;
```
We don't support in arangodb float_t/double_t. In general I think we can drop it support in iresearch too

Reply:
True. Actually, more correct would be to use 'irs::score_t'. Fixed like this.